src/Microsoft.AspNetCore.Authentication.Google/GoogleHandler.cs

```diff
@@ -87,18 +87,19 @@ public GoogleHandler(HttpClient httpClient)
         // TODO: Abstract this properties override pattern into the base class?
         protected override string BuildChallengeUrl(AuthenticationProperties properties, string redirectUri)
         {
-            var scope = FormatScope();
+            // Google Identity Platform Manual:
+            // https://developers.google.com/identity/protocols/OAuth2WebServer
 
             var queryStrings = new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase);
             queryStrings.Add("response_type", "code");
             queryStrings.Add("client_id", Options.ClientId);
             queryStrings.Add("redirect_uri", redirectUri);
 
-            AddQueryString(queryStrings, properties, "scope", scope);
-
+            AddQueryString(queryStrings, properties, "scope", FormatScope());
             AddQueryString(queryStrings, properties, "access_type", Options.AccessType);
-            AddQueryString(queryStrings, properties, "approval_prompt");
+            AddQueryString(queryStrings, properties, "prompt");
```

Comment:
Note: you'll also have to fix the Google tests: https://github.com/aspnet/Security/blob/dev/test/Microsoft.AspNetCore.Authentication.Test/Google/GoogleMiddlewareTests.cs